README.md

```diff
@@ -17,6 +17,11 @@ Infrastructure
 * [ ] Set HSTS to 31536000 (1 year)
   * `strict-transport-security: max-age=31536000`
   * [ ] If the service is not hosted under `services.mozilla.com`, it must be manually added to [Firefox's preloaded pins](https://dxr.mozilla.org/mozilla-central/source/security/manager/tools/PreloadedHPKPins.json#184). This only applies to production services, not short-lived experiments.
+* [ ] Correctly set client IP
+  * [ ] Sanitize [X-Forwarded-For](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) sent from the client. AWS and GCP's load balancers will do this automatically
```

Comment:
For AWS, worth mentioning which services i.e. do ELB classic and ALB both do this? Do we need to do this for lambda?

Reply:
Yes, both ELB and ALB do this. I think Lambda is in a different category as it will log the client ip for you and we won't have anything like iprepd-nginx in front if it.